website/settings/local-travis.py

```diff
@@ -87,3 +87,6 @@
 NEW_AND_NOTEWORTHY_LINKS_NODE = 'helloo'
 POPULAR_LINKS_NODE = 'hiyah'
 POPULAR_LINKS_REGISTRATIONS = 'woooo'
+
+EZID_USERNAME = 'testfortravisnotreal'
```

Comment:
Drive-by: Why does travis need these set? Would it make sense to just mock `request_identifiers_from_ezid` instead?